docs/swupd.1.rst

```diff
@@ -124,6 +124,10 @@ used to modify the core behavior and resources that swupd uses.
 
    Do not attempt to enforce certificate or signature checking.
 
+-  ``-n, --nosigcheck-latest``
+
+   Do not attempt to enforce certificate or signature checking on latest file.
```

Comment:
don't you think we need to add more context here to what the "latest" file is? I'm thinking this will make no sense to the common user.